cms/templates/js/team-member.underscore

```diff
@@ -16,7 +16,7 @@
       <span class="user-username"><%= user.username %></span>
       <span class="user-email">
         <a class="action action-email" href="mailto:<%= user.email %>"
-                title="<%= viewHelpers.format(gettext("send an email message to {email}"), {email: user.email})%>">
+                title="<%= viewHelpers.format(gettext("send an email message to %(email)s"), {email: user.email})%>">
```

Comment:
As part of addressing review feedback, I will change this back to the previous syntax (after discussing with @robrap).